### ISSUE TITLE
Implement gerrit listener and gerrit event coordination

### ISSUE THREAD

**Issue by erik-edling** [eiffel-community/eiffel-gerrit-plugin#11]
### Description

Dependent on #8 
Check #6 for more information of the context.

A gerrit listener should be implemented so that we can listen for certain events that the user has described in the configuration(mock the config since config is part of another story). When an event that the user want to send has been triggered it should(if possible) create a new thread where the functionality to generate and send the event is called(the actual functionality for eiffel event generation and sending is done in another story).

Is it possible to filter before getting the Gerrit message?

**Comment by m-linner-ericsson** [comment]
Would this text add more information to the issue or is it too detailed?

Input:
None

Public Interfaces:
None

Error handling:
- Not possible to create an Eiffel event
- Not possible to send the event

Internal data:
- Location of config file
- Location of where to register itself to receive Gerrit events

Data in config file:
- Database location
- RemRem address

Possible usage:
This will not be called from another class

**Comment by SantoshNC68** [comment]
should this be closed @a-breid-ericsson
